### DISCUSSION TITLE
How to obtain the field of turbulence intensity (or Tke) by using FAST.Farm?

### DISCUSSION THREAD

**Discussion by venturi123** [OpenFAST/openfast#1025] [Q&A] [answered]
Hello everyone, 

I'm working on the simulation of a wind farm with multiple turbines by using `FAST.Farm`. I wonder if the turbulence intensity or Turbulent Kinetic Energy distribution could be obtained from the `vtk_ff` folder (it seems only the velocity field can be found)  or in other ways? 

Thanks.

**Comment by jjonkman** [comment, marked as answer]
Dear @venturi123,

The TI and TKE are at specific points in the domain are not directly calculated and output from FAST.Farm.  But you can always output time series of velocity at one or more points and compute TI or TKE in a post-processing step.  The velocity time series can be obtained from the VTK visualization data (assuming the visualization time step is of sufficient resolution for accurate TI or TKE post-processing) or through the write outputs available from FAST.Farm (via FAST.Farm input parameter `NWindVel` etc.) or from the InflowWind module (via InflowWind input parameter `NWindVel` etc.) of each wind turbine OpenFAST model.

Best regards,

**Reply by venturi123** [reply]
Thanks for your clear answer！
